include/tables/sequence_table.h

```diff
@@ -0,0 +1,133 @@
+/**
+ * DEFINE_SEQUENCE(name, seqId, storageMedium, cachePolicy, seqFlags)
+ * DEFINE_SEQUENCE_PTR(seqIdReal, seqId, storageMediumReal, cachePolicyReal, seqFlags)
+ */
```

Comment:
Don't think we really need to do it now in this PR, but it would be nice if this header was in the same format as the other table headers.